data.sql

```diff
@@ -17,4 +17,27 @@ VALUES
 ('Angemon', '2005/06/12', 1, TRUE, -45),
 ('Boarmon', '2005/06/7', 7, TRUE, 20.4),
 ('Blossom', '1998/10/13', 3, TRUE, 17),
-('Ditto', '2022/05/14', 4, TRUE, 22);
\ No newline at end of file
+('Ditto', '2022/05/14', 4, TRUE, 22);
+
+INSERT INTO public.owners(full_name, age)
+VALUES 
+('Sam Smith', 34),
+('Jennifer Orwell', 19),
+('Bob', 45),
+('Melody Pond', 77),
+('Dean Winchester', 14),
+('Jodie Whittaker', 38);
+
+INSERT INTO public.species(name)
+VALUES 
+('Pokemon'), 
+('Digimon');
+
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1 WHERE species_id IS NULL;
+
+UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+UPDATE animals SET owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu');
```

Comment:
* [OPTIONAL] This insertion could have been done dynamically where you do not know the `species_id` and  `owner_id`. example:

```sql
   UPDATE animals
   SET species_id = (SELECT id from species WHERE name = 'Pokemon')
   WHERE species_id IS NULL;

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';

   UPDATE animals
   SET owner_id = (SELECT id from owners WHERE full_name = 'Jennifer Orwell')
   WHERE name = 'Gabumon' OR name = 'Pikachu';

   ...
```